File: codeforces/705A - Hulk/a.py
def main():
    n = int(input().strip())
    for idx in range(n):
        print("I hate" if idx % 2 == 0 else "I love", end="")
        print(" it" if idx == n - 1 else " that ", end="")

if __name__ == "__main__":
    main()

File: codeforces/705A - Hulk/a.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    int n;
    cin >> n;

    for(int i = 0; i < n; ++i){
        cout << (i % 2 == 0 ? "I hate" : "I love");
        cout << (i == n - 1? " it" : " that ");
    }
    return 0;
}